### PR TITLE
Replace Calypso External-link component with WordPress component

### PR DESCRIPTION
**Pull request by brezocordero**
## Description

Replace Calypso External-link component with WordPress External-link component.

### Related issue(s)

Part of #2333

### Steps to reproduce & screenshots/GIFs

The following links have been replaced:

Purchasing labels, verification address links:

|AddressVerificationLinks|
|-|

Purchasing labels, customs form, more info links:
|CustomsFormLinks|
|-|

### Checklist

- [ ] unit tests

- [ ] `changelog.txt` entry added

### CODE REVIEW

File: client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js [hunk 1]
```diff
@@ -112,12 +112,12 @@ const UnverifiedAddress = ( {
 						) }
 					</p>
 					{ uspsUrl && (
-						<ExternalLink icon={ true } href={ uspsUrl } target="_blank">
-							{ translate( 'Verify with USPS' ) }
+						<ExternalLink href={ uspsUrl } target="_blank">
+							{ translate( 'Verify with USPS ' ) }
```
Comment (brezocordero) [inline client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js:116]: Icon is included by default on WordPress component.

Comment (frosso) [reply client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js:116]: It looks like also `target="_blank"` is not necessary: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/external-link/index.js#L34

File: client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js [hunk 2]
```diff
@@ -112,12 +112,12 @@ const UnverifiedAddress = ( {
...
 						</ExternalLink>
 					) }
-					<ExternalLink icon={ true } href={ googleMapsUrl } target="_blank">
-						{ translate( 'View on Google Maps' ) }
+					<ExternalLink href={ googleMapsUrl } target="_blank">
+						{ translate( 'View on Google Maps ' ) }
```
Comment (frosso) [inline client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js:120]: I'm not seeing much of a difference with/without the blank space added here and above - in which case does it become necessary? Would it affect the translation strings?
Without space: https://d.pr/i/4P4dNY
With space: https://d.pr/i/SNat4H

**Review by frosso — APPROVED** [review]
Some minor comments, looks good otherwise

File: client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js (hunk 1 above)
Comment (brezocordero) [reply client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js]: Thanks! Removed.

File: client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js (hunk 2 above)
Comment (brezocordero) [reply client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js]: This is a mistake. Thanks for noticing. Fixed.

**Pull request merged**